pkgs/development/python-modules/jaxlib/bin.nix

```
@@ -168,10 +174,10 @@ buildPythonPackage {
 
   disabled =
```

```suggestion
  disabled = pythonOlder "3.10";
```
I am not sure that keeping an upper bound makes a ton of sense.
Actually, I would be in favor of removing the `disabled` attribute entirely.

---

I'm not aggressively trying to remove it, but it seems a bit redundant here.